recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
Hmm, I still have no idea to what values should I set `CC` and `CXX`... Isn't clang is set by default on OS X?